src/utils/isStandardSyntaxMediaFeature.js

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
@jeddy3 yes, you think create `isStandardSyntaxMediaFeature` (my rename to `isStandardSyntaxMediaFeatureName`) in this PR? or other?